.flake8

```diff
@@ -1,2 +1,6 @@
 [flake8]
-max-line-length = 120
\ No newline at end of file
+max-line-length = 120
+
+# Ignore rules from the "black" formatter. This is added
+# to avoid "Black would make changes." in houndci-bot.
+ignore = BLK, W503, W504, E121, E126, E226
```

Comment:
I see you are disabling more: showing https://pep8.readthedocs.io/en/latest/intro.html:
```
In the default configuration, the checks E121, E123, E126, E133, E226, E241, E242, E704, W503, W504 and W505 are ignored because they are not rules unanimously accepted, and PEP 8 does not enforce them
```
if you objective is to disable these too, I would disable all of them

Reply:
OK :+1: